extras/beast/test/yield_to.hpp

```diff
@@ -72,12 +72,32 @@ class enable_yield_to
         Function will be called with this signature:
 
         @code
-            void f(yield_context);
+            void f(args..., yield_context);
         @endcode
+
+        @param f The Callable object to invoke.
+
+        @param args Optional arguments forwarded to the callable object.
     */
-    template<class Function>
+#if GENERATING_DOCS
+    template<class F, class... Args>
+    void
+    yield_to(F&& f, Args&&... args);
+#else
+    template<class F>
+    void
+    yield_to(F&& f);
+
+    template<class Function, class Arg, class... Args>
     void
-    yield_to(Function&& f);
+    yield_to(Function&& f, Arg&& arg, Args&&... args)
+    {
+        yield_to(std::bind(f,
+            std::forward<Arg>(arg),
+                std::forward<Args>(args)...,
+                    std::placeholders::_1));
+    }
```

Comment:
The calls in stream.cpp both have one parameter - a lambda, so it calls the other override.